### DISCUSSION TITLE
Announce: Ai Chat — client for ChatGPT on ZeppOS

### DISCUSSION THREAD

**Discussion by melianmiko** [zepp-health/zeppos-forum#226] [General]
Unofficial free client for ChatGPT, uses OpenAI API.
What is ChatGPT? It's an AI chatbot that uses natural language processing to create humanlike conversational dialogue.

UI available in English and Russian languages.

**Supported devices:** all running ZeppOS (including Mi Band 7).
**Website:** [ZeppOS AI Chat](https://mmk.pw/en/zepp/chat/)
**Source code:** will be available later

Zepp app store moderation is pending. For now, you can install app from my website via developer mode.

![zepp_screenshot_1690376542199](https://github.com/zepp-health/zeppos-forum/assets/67851274/cff35a87-d161-4a17-8ad7-8841183028f8)
![zepp_screenshot_1690376788995](https://github.com/zepp-health/zeppos-forum/assets/67851274/4cb7807e-6876-47b7-9837-c2a179ac38e8)
![zepp_screenshot_1690376826372](https://github.com/zepp-health/zeppos-forum/assets/67851274/fff1d09d-dbdb-468c-b039-3ab70e7fe53a)
![zepp_screenshot_1690376831597](https://github.com/zepp-health/zeppos-forum/assets/67851274/94173ec7-e734-4515-9dbe-65d80d551852)
![zepp_screenshot_1690376930762](https://github.com/zepp-health/zeppos-forum/assets/67851274/ea0f3e1f-08a0-4e12-b438-6f657102e631)
![zepp_screenshot_1690377062861](https://github.com/zepp-health/zeppos-forum/assets/67851274/caf298ae-c13e-4291-92c2-95d7d361e1dd)
![zepp_screenshot_1690377530014](https://github.com/zepp-health/zeppos-forum/assets/67851274/859bd4e9-ed12-4695-9e46-02da6c011988)
![zepp_screenshot_1690376915844](https://github.com/zepp-health/zeppos-forum/assets/67851274/a135e400-6269-421c-9e86-d0103bce1a1d)

**Comment by htoooth** [comment]
@melianmiko  good job

**Comment by duanxiong** [comment]
@melianmiko Amazing and Congrats! Your app got top1 in the list of user most downloading apps for last week.
